index.html

```diff
@@ -0,0 +1,931 @@
+                    </div>
+                </a>
+                <div>
+                <div class="latestDealText">
+                    <a class="noTextDecoration">
+                        StackSkills Unlimited: Lifetime Access
+                    </a>
+                </div>
+                <br class="switch">
+                <span class="greyScale strike"> $1495 </span>$59
+                </div>
+            </div>
+            <div class="latestDealArticle8">
+                <a href="#" class="setWidth">
+                    <img class="posAbsolute" src="https://cdnp0.stackassets.com/6b0448f38732728d60f8647d2c01a68070c5faf0/store/f5b737d616d2fab13d25f07ded2660409b9f2902a13207743957dd42af48/sale_25931_primary_image.jpg" alt="Latest-Article-8">
+                    <div class="latestDealImage">
+                        <div class="alignCenter">98% OFF</div>
+                    </div>
+                </a>
+                <div>
+                <div class="latestDealText">
+                    <a class="noTextDecoration">
+                        The All-in-One Adobe Creative Cloud Suite Certification Bundle
+                    </a>
+                </div>
+                <br class="switch">
+                <span class="greyScale strike"> $1600 </span>$34
+                </div>
+            </div>
+        </section>
+    </main>
+    <footer>
+        <div class="socialIcons">
+            <div class="iconBackground">
+                <img src="Images/facebook.png" class="socialIcon" alt="Facebook Logo">
+            </div>
+            <div class="iconBackground">
+                <img src="Images/twitter.png" class="socialIcon" alt="Twitter Logo">
+            </div>
+            <div class="iconBackground">
+                <img src="Images/white-youtube-logo-transparent-24.png" class="socialIcon" alt="Youtube Logo">
+            </div>
+            <div class="iconBackground">
+                <img src="Images/instagram.png" class="socialIcon" alt="Instagram Logo">
+            </div>
+            <div class="iconBackground">
+                <img src="Images/pintrest.png" class="socialIcon" alt="Pintrest Logo">
+            </div>
+            <div class="iconBackground">
+                <img src="Images/feed.png" class="socialIcon" alt="Feed Logo">
+            </div>
+            <div class="iconBackground">
+                <img src="Images/email.png" class="socialIcon" alt="Email Logo">
+            </div>
+        </div>
+        <div class="importantLinks">
+            <p>EVENTS</p> 
+            <p>ABOUT</p> 
+            <p>ADVERTISE</p> 
+            <p>JOBS</p> 
+            <p>CONTACT</p> 
+        </div>
+        <div class="footerSecond">
+            <a href="#"><span class="colorWhite">The Next Web </span></a> 
+            <span class="greyScale">© 2006–2020 The Next Web B.V. Made with ♥ in Amsterdam. Powered by </span> 
+            <a href="#"><span class="colorWhite">STACKPATH</span></a>
+        </div>
```

Comment:
All the elements in the footer should be links. In the first half you have images, enclose them in `<a>` tags.
In the second part you have `<p>` tags, change them to be `<a>` tags.
This is the best practice when building a footer and you want each element to redirect to another place.